libraries/Wire/src/utility/twi.c

```diff
@@ -233,7 +243,12 @@ uint8_t twi_writeTo(uint8_t address, uint8_t* data, uint8_t length, uint8_t wait
   }
 
   // wait until twi is ready, become master transmitter
+  twi_timeout(1);
   while(TWI_READY != twi_state){
+    if (twi_timeout(0)) 
+	{
```

Comment:
Please follow the formatting style of the rest of the code in the file and attach this brace to the if statement.